frontend/src/App.jsx

```diff
@@ -1,19 +1,15 @@
+import { Outlet } from "react-router-dom";
 import "./App.css";
 import React, { useState } from "react";
 import Navbar from "./components/navbar/Navbar";
-import CardList from "./components/CardList/CardList";
 import Footer from "./components/footer/Footer";
-import Filtres from "./components/filtres/Filtres";
 
 function App() {
   const [basketCount, setBasketCount] = useState(0);
   return (
     <div className="App">
       <Navbar basketCount={basketCount} />
-      <div className="main">
-        <Filtres />
-        <CardList basketCount={basketCount} setBasketCount={setBasketCount} />
-      </div>
+      <Outlet setBasketCount={setBasketCount} />
```

Comment:
Never pass a props to the Outlet (unless you really know what you're doing) , your App.jsx is not designed to handle states.